### PR TITLE
Add 'set_switch_override' text.

### PR DESCRIPTION
**Pull request by GeoffAtHome**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- [Link to parent pull request in the codebase: ](https://github.com/home-assistant/core/pull/45558)

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.

- [x ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/geniushub.markdown [hunk 1]
```diff
@@ -210,25 +213,25 @@ geniushub:
...
+description: Your Genius Hub username.
+required: false
+type: string
 password:
-  description: Your Genius Hub password.
-  required: false
-  type: string
+description: Your Genius Hub password.
+required: false
+type: string
```
Comment (frenck) [inline source/_integrations/geniushub.markdown]: This should not be formatted like this.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/geniushub.markdown (hunk 1 above)
Comment (GeoffAtHome) [reply source/_integrations/geniushub.markdown]: VS code is removing the leading/excess white space. I need to see how I can fix this.

Comment (GeoffAtHome) [reply source/_integrations/geniushub.markdown]: Changed VS code setting.

Comment (GeoffAtHome) [reply source/_integrations/geniushub.markdown]: OK.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — CHANGES_REQUESTED** [review]
Oh, not approved. There is still an issue. The PR is targeting the `current` branch. However, since this documentation on a code change, it depends on a next release of Home Assistant.

Thus, this PR should be targeting the `next` branch. Please change the target branch of this PR and run a git rebase if needed.

**Comment by GeoffAtHome** [comment]
Rebase still to be done.

**Comment by GeoffAtHome** [comment]
Rebase to next done.

**Comment by GeoffAtHome** [comment]
I think I have changed to  `home-assistant:next `.  If this is wrong I need assistance here as I am unsure what needs to be done.

**Comment by frenck** [comment]
Yes, the target is adjusted, but your upstream is based on current, thus there are unrelated commits in your PR.

You need to preform a git rebase.

**Comment by GeoffAtHome** [comment]
I ran:
```
git fetch upstream next  
git rebase upstream/next
git push origin --force-with-lease
git push origin --force
```

Do I need to do anything else?

**Comment by klaasnicolaas** [comment]
Thought the rebase was resolved but still seeing unrelated commits in the pull request.

**Comment by frenck** [comment]
@GeoffAtHome 

```bash
git rebase upstream/next YOUR_BRANCH~2
git log
git push --force
```

The git log is there for you to check if everything looks good now.

**Comment by GeoffAtHome** [comment]
I tried:
git rebase upstream/next branch
git log

The log has so many entries I did not know what was good or not.

I delete the fork and tried again.
I think this is right now. The change is only add one line.

**Comment by GeoffAtHome** [comment]
I am not sure why I now have an unknown repository.
GeoffAtHome wants to merge 4 commits into **home-assistant:next** from **unknown repository**

It may be simpler to delete this PR and start again. Let me know what you think.

**Comment by frenck** [comment]
You deleted the fork, and therefore the reference of this PR.
This can only be fixed by creating a new PR, therefore, closing this one.

**Pull request closed without merging**